mkdocs.1

```diff
@@ -0,0 +1,27 @@
+.TH "MKDOCS" "1" "0.1" "William Moreno Reyes" ""
+.SH "NAME"
+mkdocs \- Create HTML documentation from markdown sources
+.SH "SYNOPSIS"
+mkdocs [help|new|build|serve|gh\-deploy] {options}
```

Comment:
`mkdocs help` doesn't work. This is incorrect on mkdocs.org I see. This would be correct

    mkdocs [build|gh\-deploy|new|serve] {options}